domain/src/main/java/org/oppia/domain/oppialogger/exceptions/ExceptionsController.kt

```diff
@@ -10,103 +10,81 @@ import org.oppia.util.data.DataProviders
 import org.oppia.util.logging.ConsoleLogger
 import org.oppia.util.logging.ExceptionLogger
 import org.oppia.util.networking.NetworkConnectionUtil
-import org.oppia.util.system.OppiaClock
 import javax.inject.Inject
 
+const val EXCEPTIONS_CONTROLLER = "Exceptions Controller"
+
+/** Controller for handling exception logging. */
 class ExceptionsController @Inject constructor(
   private val exceptionLogger: ExceptionLogger,
   private val dataProviders: DataProviders,
   cacheStoreFactory: PersistentCacheStore.Factory,
   private val consoleLogger: ConsoleLogger,
   private val networkConnectionUtil: NetworkConnectionUtil,
-  private val oppiaClock: OppiaClock,
   @ExceptionLogStorageCacheSize private val exceptionLogStorageCacheSize: Int
 ) {
   private val exceptionLogStore =
     cacheStoreFactory.create("exception_logs", OppiaExceptionLogs.getDefaultInstance())
 
-  lateinit var exceptionCause: Throwable
+  /** Logs a NON-FATAL exception. */
```

Comment:
How is the timestamp used? That detail seems missing in this & the next method's KDoc. Please update.

Reply:
Done.